ucm2/USB-Audio/Realtek/ALC4080-HiFi.conf

```diff
@@ -0,0 +1,57 @@
+SectionDevice."Speaker" {
+	Comment "Speakers"
+	Value {
+		PlaybackChannels 8
+		PlaybackPriority 200
+		PlaybackPCM "hw:${CardId}"
+		JackControl "Speaker - Output Jack"
+		PlaybackMixerElem "Speaker"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Front Headphones"
+	Value {
+		PlaybackPriority 300
+		PlaybackPCM "hw:${CardId},1"
+		JackControl "Headphone - Output Jack"
+		PlaybackMixerElem "Front Headphone"
+	}
+}
+
+SectionDevice."SPDIF" {
+	Comment "S/PDIF Out"
+	Value {
+		PlaybackPriority 100
+		PlaybackPCM "hw:${CardId},3"
+		PlaybackMixerElem "IEC958"
+	}
+}
+
+SectionDevice."Line" {
+	EnableSequence [
+		cset "name='Line Capture Switch' on"
+	]
+
+	Comment "Line In"
+	Value {
+		CapturePriority 100
+		CapturePCM "hw:${CardId},1"
```

Comment:
Detects plug in, but have not yet got any sound in. Will try with a better device soon.

Reply:
Line-in works well enough. I tested with my phone connected though I had to turn up the volume a bit.

It may be a good thing that there is not a boost like Windows. This allows more headroom. OBS can boost the source live (Audacity cannot). When I use a lavalier mic in normal position and boost with OBS to 20 dBm, it sounds good enough and there is not much noise.